README.md

```diff
@@ -551,6 +549,8 @@ extend (whether to other areas of Astronomy or in other domains).
 Release History
 ---------------
 
+4.9.1: 01 August 2017 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.825839.svg)](https://doi.org/10.5281/zenodo.825839)
```

Comment:
The badge at the top of the file will also need updating with this info to, I believe.